### PR TITLE
feat(chart,calendar):  two grammar corrections

### LINKED ISSUE #7196: `lv_calendar_set_showed_date()` Contains a Grammar Error
### LVGL version

v9.3

### What happened?

`lv_calendar_set_showed_date()` -- to be correct English -- should be `lv_calendar_set_shown_date()`, or even better (reflecting what the C code does in the function): `lv_calendar_set_month_shown()` (since the day number is hard-coded to 1 and it only accepts `year` and `month` arguments).

Current documentation contains this:
```
lv_calendar_set_shown_date(calendar, year, month)
```

I found this because it was partially corrected in the documentation, but not in the code.  I don't know which was first, but can attest that *at least* the code that needs to change, and the documentation as well if `lv_calendar_set_month_shown()` is selected instead of what is in the current documentation.  Perhaps this can be done with a back-patch using a macro so that existing apps are not broken?

And the brief description in the API documentation should change from:
```
Set the currently showed
```
to
```
Set month currently shown
```

### How to reproduce?

See above.

### PR DESCRIPTION
**Pull request by vwheeler63**
- lv_calendar_set_showed_date() => lv_calendar_set_month_shown()
- lv_chart_set_all_value() =>  lv_chart_set_all_values()

Both of these were discussed in Issue #7196 and guidance given by @kisvegabor therein so as to not break the API.

Resolves #7196

cc:  @kisvegabor 

### Notes
- Update the [Documentation](https://github.com/lvgl/lvgl/tree/master/docs) if needed.  n/a
- Add [Examples](https://github.com/lvgl/lvgl/tree/master/examples) if relevant.  n/a
- Add [Tests](https://github.com/lvgl/lvgl/blob/master/tests/README.md) if applicable.  n/a
- If you added new options to `lv_conf_template.h` run [lv_conf_internal_gen.py](https://github.com/lvgl/lvgl/blob/master/scripts/lv_conf_internal_gen.py) and update [Kconfig](https://github.com/lvgl/lvgl/blob/master/Kconfig).  n/a
- Run `scripts/code-format.py` (`astyle v3.4.12` needs to installed by running `cd scripts; ./install_astyle.sh`) and follow the [Code Conventions](https://docs.lvgl.io/master/CODING_STYLE.html).  Done.
- Mark the Pull request as [Draft](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request) while you are working on the first version, and mark is as _Ready_ when it's ready for review.  Done.
- When changes were requested, [re-request review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) to notify the maintainers.  Done.
- Help us to review this Pull Request! Anyone can [approve or request changes](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/approving-a-pull-request-with-required-reviews).

### CODE REVIEW

**Comment by vwheeler63** [comment]
Hi, @kisvegabor !

FYI, having
```c
static void lv_calendar_set_showed_date(lv_obj_t * obj, uint32_t year, uint32_t month)
{
    lv_calendar_set_month_shown(obj, year, month);
}
```
in `lv_api_map_v9_1.h` was causing 4 of the CI/CD checks (with compilations) to fail with the message 
```
Error: /home/runner/work/lvgl/lvgl/src/draw/vg_lite/../../../src/lv_api_map_v9_1.h:30:13: error: ‘lv_calendar_set_showed_date’ defined but not used [-Werror=unused-function]
```
So I changed it to 
```c
#define lv_calendar_set_showed_date lv_calendar_set_month_shown
```
and that seems to be working.

File: src/lv_api_map_v9_1.h [hunk 1]
```diff
@@ -27,6 +27,11 @@ extern "C" {
  * GLOBAL PROTOTYPES
  **********************/
 
+static inline void lv_calendar_set_showed_date(lv_obj_t * obj, uint32_t year, uint32_t month)
```
Comment (kisvegabor) [inline src/lv_api_map_v9_1.h]: Can it be a `define` similar to the others below?

**Review by kisvegabor — COMMENTED** [review]
It looks good in general, thank you!

File: src/lv_api_map_v9_1.h (hunk 1 above)
Comment (vwheeler63) [reply src/lv_api_map_v9_1.h]: > Can it be a `define` similar to the others below?

Done!  😄

**Comment by lhdjply** [comment]
Change the last line of the file [src/lv_api_map_v9_1.h ](https://github.com/lvgl/lvgl/blob/36695d95ab94b6ab5ed6d405845a1f6177f3f188/src/lv_api_map_v9_1.h#L92)to LV_API_MAP_V9_1_H.

**Comment by vwheeler63** [comment]
> Can you change the last line of the file [src/lv_api_map_v9_1.h ](https://github.com/lvgl/lvgl/blob/36695d95ab94b6ab5ed6d405845a1f6177f3f188/src/lv_api_map_v9_1.h#L92)to LV_API_MAP_V9_1_H？

Good eye!  Done!

**Review by kisvegabor — DISMISSED** [review]

**Review by liamHowatt — COMMENTED** [review]
Please change the instances of `lv_calendar_set_showed_date` in test_calendar.c and calendar.rst.

Please change the `lv_chart_set_all_value` in chart.rst.

**Comment by vwheeler63** [comment]
> Please change the instances of `lv_calendar_set_showed_date` in test_calendar.c and calendar.rst.
> 
> Please change the `lv_chart_set_all_value` in chart.rst.

Done!  I held these back originally because at the time this was submitted, I think they were pending in a PR with major re-work, so I didn't want to cause a conflict.  They are contained herein now.

**Review by kisvegabor — APPROVED** [review]

**Review by liamHowatt — APPROVED** [review]
> Done! I held these back originally because at the time this was submitted, I think they were pending in a PR with major re-work, so I didn't want to cause a conflict. They are contained herein now.

Oops, thanks for changing it. No worries next time if it's coming later. I think merge conflicts won't occur if the change is identical in both histories.

`test_calendar.c` still uses `lv_calendar_set_showed_date` but I will merge now since it's not critical.

**Comment by vwheeler63** [comment]
> `test_calendar.c` still uses `lv_calendar_set_showed_date` but I will merge now since it's not critical.

Thank you, and ouch!  My string scan was not wide enough.  Noted for next time.  Correction submitted in PR #7375 .

**Pull request merged**